src/App.js

```diff
@@ -1,5 +1,175 @@
+import { Console, Random } from "@woowacourse/mission-utils";
+
+const ERROR_MESSAGE = {
+    INVALID_CAR_NAME: "[ERROR] 자동차 이름은 5자 이하 한글, 영문만 입력 가능합니다.",
+    INVALID_CAR_COUNT: "[ERROR] 자동차는 2대 이상 입력해주세요.",
+    DUPLICATED_CAR_NAME: "[ERROR] 중복된 자동차 이름이 존재합니다.",
+    INVALID_TRY_COUNT: "[ERROR] 시도 횟수는 1 이상 10 이하 정수값을 입력해주세요.",
+    INVALID_NUMBER: "[ERROR] 시도 횟수는 숫자로 입력해주세요.",
+};
+
 class App {
-  async run() {}
+    racingCars = {};
+
+    initializeRacingCars(carNames) {
+        carNames.forEach((carName) => {
+            this.racingCars[carName] = 0;
+        });
+    }
+
+    saveRacingCarPosition(carName, position) {
+        this.racingCars[carName] = position;
+    }
+
+    getRacingCarPosition(carName) {
+        return this.racingCars[carName];
+    }
+
+    getRacingCarNames() {
+        return Object.keys(this.racingCars);
+    }
+
+    splitCarNamesStr(carNamesStr) {
+        const carNames = carNamesStr.trim().split(",");
+        return carNames.map((carName) => carName.trim());
+    }
+
+    isValidCarName(carName) {
+        const regExp = /^[a-z|A-Z|가-힣]+$/; // 한글, 영문만 입력 가능
+
+        if (carName.length > 5) {
+            return false;
+        }
+
+        if (!carName.match(regExp)) {
+            return false;
+        }
+
+        return true;
+    }
+
+    hasUniqueCarNames(carNames) {
+        const uniqueCarNames = new Set(); // 중복된 자동차 이름 체크
+
+        carNames.forEach((carName) => {
+            uniqueCarNames.add(carName.toLowerCase());
+        });
+
+        if (uniqueCarNames.size !== carNames.length) {
+            return false;
+        }
+
+        return true;
+    }
+
+    isValidCarCount(carNames) {
+        if (carNames.length < 2) {
+            return false;
+        }
+
+        return true;
```

Comment:
```suggestion
        return carNames.length >= 2;
```

간단하게 이런 식으로 변경할 수도 있을 것 같아요! 비슷하게 조건문으로 boolean 값을 반환하는 부분들은 모두 바꿔볼 수 있겠네요 :)

Reply:
코드가 간단해지고 의도 파악도 더욱 수월한 것 같네요.
감사합니다!

Reply:
코드가 간단해지고 의도 파악도 더욱 수월한 것 같네요.
감사합니다!